src/amber/dsl/server.cr

```diff
@@ -1,15 +1,17 @@
-module Amber::DSL
-  module Server
-    macro routes(valve, scope = "")
-      router.draw {{valve}}, {{scope}} do
-        {{yield}}
-      end
+module Amber::DSL::Server
+  macro routes(valve, scope = "")
+    router.draw {{valve}}, {{scope}} do
+      {{yield}}
     end
+  end
 
-    macro pipeline(valve)
-      handler.build {{valve}} do
-        {{yield}}
-      end
+  macro pipeline(valve)
+    handler.build {{valve}} do
+      {{yield}}
     end
   end
+
+  macro load_environment
```

Comment:
This seems that belongs more in the CLI than in the server. We can pair on this for clarification